.circleci/config.yml

```diff
@@ -375,13 +375,13 @@ commands:
             mkdir -p /tmp/junit
             ng test $ANGULAR_PROJECT_NAME --watch=false --browsers ChromeHeadlessNoSandbox --reporters junit
           environment:
-            JUNIT_REPORT_PATH: /tmp/junit/
+            JUNIT_REPORT_PATH: /tmp/ui/junit/
```

Comment:
`/tmp/ui/junit/` so that ui test result file is not overriden by sdk & toolkit test result file.